src/RA_Dlg_MemBookmark.cpp

```diff
@@ -799,27 +774,25 @@ void Dlg_MemBookmark::ImportFromFile(std::wstring sFilename)
     {
         ClearAllBookmarks();
 
-        const auto& BookmarksData{ doc["Bookmarks"] };
+        const auto& BookmarksData{doc["Bookmarks"]};
+
         for (const auto& bmData : BookmarksData.GetArray())
         {
-            auto NewBookmark{ new MemBookmark };
-            NewBookmark->SetDescription(ra::Widen(bmData["Description"].GetString()));
+            MemBookmark NewBookmark;
```

Comment:
The darn new was the source of all our problems. But there were many problems with this class. Like casting that could have been avoided, algorithm complexity, raw pointers, etc.